### DISCUSSION TITLE
Sigma Convertor Always adds quotation marks

### DISCUSSION THREAD

**Discussion by OmerTirosh** [Security-Onion-Solutions/securityonion#3977] [Unsupported Versions]
Hi, 

Sigmac always adds quotation marks, 
When a new rule is written in the Playbook.

sigmac always adds quotation marks to the esquery, regardless to your Sigma rule, it's should only add them when you put them in your rule.
 
String inside quotation marks parse the data with the analyzer and than search with the output tokens, therefore you cannot use wildcard in your alert (because the security analyzer will remove them) and part of the built in rules won't work.

Thanks!!

**Comment by defensivedepth** [comment]
@OmerTirosh If this is still an issue, can you please provide an example?
